src/frontend/screens/Game/GamePage/index.css

```diff
@@ -494,7 +494,7 @@
 
     .gameTools .link {
       width: 100%;
-      padding: 0 0 0.5rem;
+      padding: 0.3rem;
```

Comment:
I think we can just remove this line, this element already has a padding for the `.button.is-link` that sets `0.25em`